### PR TITLE
black: m-w

### PR DESCRIPTION
**Pull request by LeoMcA**

### CODE REVIEW

File: kitsune/sumo/tests/test_parser.py [hunk 1]
```diff
@@ -352,8 +339,7 @@ def test_link_hash_name_markup(self):
         """Internal link with hash and name."""
         text = "[[Installing Firefox#section 3|this name]]"
         eq_(
-            '<p><a href="/en-US/kb/installing-firefox#section_3"'
-            + ">this name</a>\n</p>",
+            '<p><a href="/en-US/kb/installing-firefox#section_3"' + ">this name</a>\n</p>",
```
Comment (akatsoulas) [inline kitsune/sumo/tests/test_parser.py:342]: Can be merged in a single string

File: kitsune/upload/tests/test_views.py [hunk 2]
```diff
@@ -117,73 +116,73 @@ def test_delete_image_owner(self):
...
         json_r = json.loads(r.content)
-        eq_('error', json_r['status'])
-        eq_('Invalid or no image received.', json_r['message'])
+        eq_("error", json_r["status"])
+        eq_("Invalid or no image received.", json_r["message"])
         assert (
-            json_r['errors']['image'][0] ==
-            "File extension 'tiff' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'.")
+            json_r["errors"]["image"][0]
+            == "File extension 'tiff' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'."  # noqa
```
Comment (akatsoulas) [inline kitsune/upload/tests/test_views.py:184]: we can remove noqa from here

File: kitsune/users/tasks.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def process_event_subscription_state_change(event_id):
     last_event = AccountEvent.objects.filter(
         profile_id=event.profile.pk,
         status=AccountEvent.PROCESSED,
-        event_type=AccountEvent.SUBSCRIPTION_STATE_CHANGE
+        event_type=AccountEvent.SUBSCRIPTION_STATE_CHANGE,
```
Comment (akatsoulas) [inline kitsune/users/tasks.py:27]: nit: I wouldn't go for the `,`

File: kitsune/users/models.py [hunk 4]
```diff
@@ -320,38 +329,38 @@ def get_profile(u):
...
     @classmethod
     def get_for_user(cls, user, name):
         from kitsune.users.forms import SettingsForm
+
         form = SettingsForm()
         if name not in list(form.fields.keys()):
-            raise KeyError(("'{name}' is not a field in "
-                            "user.forms.SettingsFrom()").format(name=name))
+            raise KeyError(
+                ("'{name}' is not a field in " "user.forms.SettingsFrom()").format(name=name)
```
Comment (akatsoulas) [inline kitsune/users/models.py:358]: can be merged into a single string

File: kitsune/wiki/admin.py [hunk 5]
```diff
@@ -31,34 +45,35 @@ def _set_archival(queryset, should_be_archived):
...
         phrase = (
-            'document, along with its English version or translations, was '
-            'marked as '
-            if count == 1 else
-            'documents, along with their English versions or translations, '
-            'were marked as ')
-        self.message_user(request, '%s %s %s.' % (count, phrase, verb))
+            "document, along with its English version or translations, was " "marked as "
+            if count == 1
+            else "documents, along with their English versions or translations, " "were marked as "
```
Comment (LeoMcA) [inline kitsune/wiki/admin.py:50]: weird concatenations

File: kitsune/wiki/config.py [hunk 6]
```diff
@@ -53,8 +50,7 @@
 
 SIGNIFICANCES_HELP = {
     TYPO_SIGNIFICANCE: _lazy(
-        "These minor changes are not important for localizers and "
-        "they will not be notified."
+        "These minor changes are not important for localizers and " "they will not be notified."
```
Comment (LeoMcA) [inline kitsune/wiki/config.py:53]: String concat

File: kitsune/wiki/events.py [hunk 7]
```diff
@@ -334,16 +318,12 @@ def _make_mail(locale, user, watches):
                 c["watch"] = watches[0]
                 c["reviewer"] = revision.reviewer
 
-                subject = _(
-                    "{title} ({locale}) has a new approved " "revision ({reviewer})"
-                )
+                subject = _("{title} ({locale}) has a new approved " "revision ({reviewer})")
```
Comment (LeoMcA) [inline kitsune/wiki/events.py:321]: string literal concatenation

File: kitsune/wiki/parser.py [hunk 8]
```diff
@@ -448,8 +439,7 @@ def _hook_template(self, parser, space, title):
         template_title = "Template:" + short_title
 
         message = (
-            _('The template "%s" does not exist or has no approved ' "revision.")
-            % short_title
+            _('The template "%s" does not exist or has no approved ' "revision.") % short_title
```
Comment (LeoMcA) [inline kitsune/wiki/parser.py:442]: string literal concat

File: kitsune/wiki/tasks.py [hunk 9]
```diff
@@ -101,27 +108,26 @@ def send_contributor_notification(based_on_ids: List[int], revision_id: int, doc
...
+    }
 
     msgs = []
 
     @email_utils.safe_translation
     def _make_mail(locale, user):
         if revision.is_approved:
-            subject = _('A revision you contributed to has '
-                        'been approved: {title}')
+            subject = _("A revision you contributed to has " "been approved: {title}")
```
Comment (LeoMcA) [inline kitsune/wiki/tasks.py:128]: string literal concat

File: kitsune/wiki/tasks.py [hunk 10]
```diff
@@ -101,27 +108,26 @@ def send_contributor_notification(based_on_ids: List[int], revision_id: int, doc
...
         else:
-            subject = _('A revision you contributed to has '
-                        'been reviewed: {title}')
+            subject = _("A revision you contributed to has " "been reviewed: {title}")
```
Comment (LeoMcA) [inline kitsune/wiki/tasks.py:130]: string literal concat

File: kitsune/wiki/tests/test_models.py [hunk 11]
```diff
@@ -210,58 +222,62 @@ def test_majorly_outdated_with_unapproved_parents(self):
...
         # ...and a translation with a revision based on nothing.
-        trans = DocumentFactory(parent=parent_rev.document, locale='de')
+        trans = DocumentFactory(parent=parent_rev.document, locale="de")
         trans_rev = RevisionFactory(document=trans, is_approved=True)
 
-        assert trans_rev.based_on is None, \
-            ('based_on defaulted to something non-None, which this test '
-             "wasn't expecting.")
+        assert trans_rev.based_on is None, (
+            "based_on defaulted to something non-None, which this test " "wasn't expecting."
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_models.py:229]: string literal concat

File: kitsune/wiki/tests/test_parser.py [hunk 12]
```diff
@@ -25,132 +35,147 @@ def doc_rev_parser(*args, **kwargs):
...
+        doc = pq(p.parse("{warning}\nthis is a warning\n{/warning}"))
+        eq_("this is a warning", doc("div.warning").text())
 
     def test_warning_multiline_breaks(self):
         """Multiline breaks warning syntax"""
         p = WikiParser()
-        doc = pq(p.parse('\n\n{warning}\n\nthis is a warning\n\n'
-                         '{/warning}\n\n'))
-        eq_('this is a warning', doc('div.warning').text())
+        doc = pq(p.parse("\n\n{warning}\n\nthis is a warning\n\n" "{/warning}\n\n"))
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_parser.py:67]: string literal concat

File: kitsune/wiki/tests/test_parser.py [hunk 13]
```diff
@@ -25,132 +35,147 @@ def doc_rev_parser(*args, **kwargs):
...
+            '<span class="key">"Param-One" and</span> + <span class="key">'
             'param</span> + <span class="key">two</span>',
-            key_p.sub(_key_split, '{key  "Param-One" and + param+two}'))
-        eq_('<span class="key">multi\nline</span> + '
-            '<span class="key">me</span>',
-            key_p.sub(_key_split, '{key multi\nline\n+me}'))
+            key_p.sub(_key_split, '{key  "Param-One" and + param+two}'),
+        )
+        eq_(
+            '<span class="key">multi\nline</span> + ' '<span class="key">me</span>',
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_parser.py:128]: string literal concat

File: kitsune/wiki/tests/test_parser.py [hunk 14]
```diff
@@ -25,132 +35,147 @@ def doc_rev_parser(*args, **kwargs):
...
 
     def test_key_split_brace_callback(self):
         """Adding brace inside {key ...}"""
         key_p = PATTERNS[2][0]
-        eq_('<span class="key">ctrl</span> + <span class="key">and</span> '
-            'Here is }',
-            key_p.sub(_key_split, '{key ctrl + and} Here is }'))
-        eq_('<span class="key">ctrl</span> + <span class="key">and</span> + '
+        eq_(
+            '<span class="key">ctrl</span> + <span class="key">and</span> ' "Here is }",
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_parser.py:136]: string literal concat

File: kitsune/wiki/tests/test_parser.py [hunk 15]
```diff
@@ -522,126 +558,135 @@ def test_block(self):
...
     def test_late_close(self):
         """If the closing for tag is not closed by the time the enclosing
         element of the opening for tag is closed, close the for tag
         just before the enclosing element."""
         self.assertWikiHtmlEqual(
-            '*{for}One\n*Fish\n\nTwo\n{/for}',
-            '<ul><li><span class="for">One</span></li>'
-            '<li>Fish</li></ul><p>Two</p>')
+            "*{for}One\n*Fish\n\nTwo\n{/for}",
+            '<ul><li><span class="for">One</span></li>' "<li>Fish</li></ul><p>Two</p>",
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_parser.py:616]: string literal concat

File: kitsune/wiki/tests/test_templates.py [hunk 16]
```diff
@@ -678,8 +674,7 @@ def test_new_document_POST_invalid_category(self):
         ul = doc("#document-form > ul.errorlist")
         eq_(1, len(ul))
         assert (
-            "Select a valid choice. 963 is not one of the available "
-            "choices." in ul("li").text()
+            "Select a valid choice. 963 is not one of the available " "choices." in ul("li").text()
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_templates.py:677]: string literal concat

File: kitsune/wiki/tests/test_utils.py [hunk 17]
```diff
@@ -75,58 +77,47 @@ def test_num_active_contributors(self):
...
 
 
-@override_settings(BITLY_LOGIN='test', BITLY_API_KEY='test-apikey')
+@override_settings(BITLY_LOGIN="test", BITLY_API_KEY="test-apikey")
 class GenerateShortUrlTestCase(TestCase):
-
     def setUp(self):
-        self.test_url = ('https://support.mozilla.org/en-US/kb/'
-                         'update-firefox-latest-version')
+        self.test_url = "https://support.mozilla.org/en-US/kb/" "update-firefox-latest-version"
```
Comment (LeoMcA) [inline kitsune/wiki/tests/test_utils.py:91]: string literal concat

File: kitsune/search/views.py [hunk 18]
```diff
@@ -151,67 +148,68 @@ def simple_search(request):
...
     cleaned = search_form.cleaned_data
 
-    language = locale_or_default(cleaned['language'] or request.LANGUAGE_CODE)
-    lang_name = settings.LANGUAGES_DICT.get(language.lower()) or ''
+    language = locale_or_default(cleaned["language"] or request.LANGUAGE_CODE)
+    lang_name = settings.LANGUAGES_DICT.get(language.lower()) or ""
 
     searcher = generate_simple_search(search_form, language, with_highlights=True)
-    searcher = searcher[:settings.SEARCH_MAX_RESULTS]
+    searcher = searcher[: settings.SEARCH_MAX_RESULTS]
```
[the remark this answers is not in the document]
Comment (LeoMcA) [reply kitsune/search/views.py:171]: As discussed on slack, it is

File: kitsune/upload/tests/test_views.py (hunk 2 above)
Comment (LeoMcA) [reply kitsune/upload/tests/test_views.py:184]: I don't think we can, the line is longer than 99 chars without it

**Pull request merged**